TenureInformationApi/V1/Boundary/Requests/CreateTenureRequestObject.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TenureInformationApi.V1.Domain;
+
+namespace TenureInformationApi.V1.Boundary.Requests
+{
+    public class CreateTenureRequestObject
```

Comment:
There is currently no validation on any of the request object data, but should there be?

Reply:
I don't see any mention of it, but if anyone mentions any validation then I'm happy to add it in

Reply:
I've added Validation and relevant tests